operator/pkg/cli/migrate_v2.go

```diff
@@ -0,0 +1,70 @@
+package cli
+
+import (
+	"fmt"
+	"log"
+
+	ecv1beta1 "github.com/replicatedhq/embedded-cluster/kinds/apis/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/operator/pkg/cli/migratev2"
+	"github.com/replicatedhq/embedded-cluster/operator/pkg/k8sutil"
+	"github.com/spf13/cobra"
+)
+
+// MigrateV2Cmd returns a cobra command for migrating the installation from v1 to v2.
+func MigrateV2Cmd() *cobra.Command {
+	var installationFile, licenseSecret, appVersionLabel string
+
+	var installation *ecv1beta1.Installation
+
+	cmd := &cobra.Command{
+		Use:          "migrate-v2",
+		Short:        "Migrates the Embedded Cluster installation from v1 to v2",
+		SilenceUsage: true,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			in, err := getInstallationFromFile(installationFile)
+			if err != nil {
+				return fmt.Errorf("failed to get installation from file: %w", err)
+			}
+			installation = in
```

Comment:
should `runtimeconfig.Set(installation.Spec.RuntimeConfig)` be called after this?

Reply:
fixed and added a note